#!
#! /usr/bin/env python

PACKAGE='group3_lab7'
import roslib
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()
# Name, Type, Reconfiguration level, Description, Default, Min, Max
gen.add("lookahead_distance", double_t, 0, "Lookahead distance.", 4.0, 0.2, 5.0)
gen.add("forward_direction", bool_t, 0, "Follow the path in the forward direction.", True)

exit(gen.generate(PACKAGE, "group3_lab7", "path_publisher_"))